# Vernier GoDirect Hand Dynamometer configuration
# Dynamometer name and ID
device_name: 'GDX-HD 151025S9'
# Select sensor to sample (works only for 1 - Force)
# 1: Force (N)
# 2: X-axis acceleration (m/s²)
# 3: Y-axis acceleration (m/s²)
# 4: Z-axis acceleration (m/s²)
# 5: X-axis gyro (rad/s)
# 6: Y-axis gyro (rad/s)
# 7: Z-axis gyro (rad/s)
selected_sensor: 1
# Measurement type: pinch/grip
measurement_type: 'grip'
# Dynamometer sampling rate (max 1000 Hz)
# Sampling period is integer in ms, so possible rates are:
# 1000, 500, 333.33, 200, 166.66, 142.857, 125, 111, 100, ...
sampling_rate: 500 # Hz
# Use average of first several sec to zero the response (0 - no calibration).
# Hold the sensor in same position it will be used for data collection.
# Do not grip sensor during this time.
calibrate_signal: 1 # sec
# Size of the queue for asynchronous publishing on topic
queue_size: 10